docs/pages/2019_MARVEL_Psik_MaX/notebooks/bandstructure.ipynb

```diff
@@ -138,6 +153,31 @@
     "    **results['seekpath_parameters'].get_dict()))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the pseudopotential family that you downloaded is the same efficiency pseudopotential family, but an older version  (the https://archive.materialscloud.org/file/2018.0001/v1/SSSP_efficiency_pseudos.tar.gz -- note the v1 in the link -- you can use the PwBandStructureWorkChain can be used, by providing a modification of the existing protocol"
```

Comment:
I would say more something like:

If you want to use a different pseudopotential family (or version of the family) (for instance [SSSP v1.0](https://archive.materialscloud.org/file/2018.0001/v1/SSSP_efficiency_pseudos.tar.gz) instead of the default SSSP v1.1) you can pass an additional parameter when calling the WorkChain, as follows: 
```
   protocol = Dict(dict={'name':'theos-ht-1.0', 'modifiers': {'pseudo' : 'SSSP-efficiency-1.0'}})
```
(note that only some values are accepted for `pseudo`, that you can find [here](https://github.com/aiidateam/aiida-quantumespresso/blob/b02250146576eb573ccb45d05047075f54853f9d/aiida_quantumespresso/utils/protocols/pw.py#L24)).